.buildkite/Dockerfile-release

```diff
@@ -0,0 +1,5 @@
+FROM debian:bookworm-slim
+
+ADD https://github.com/buildkite/github-release/releases/download/v1.0/github-release-linux-amd64 /usr/local/bin/github-release
```

Comment:
at some point we should really switch away from `github-release`. The repo in our GH org is archived and we're not maintaining it.

However, for now I'm looking to get this release pipeline back to working the way it was. I'm not super keen to overhaul the tooling.